pkg/kubectl/local.go

```diff
@@ -52,12 +52,13 @@ func outputMatrix(cmd *exec.Cmd) (stdout, stderr, combined string, err error) {
 	stdoutPipe, _ := cmd.StdoutPipe()
 	stderrPipe, _ := cmd.StderrPipe()
 
-	stdoutWriter := io.MultiWriter(&combinedBuf, &stdoutBuf)
-	stderrWriter := io.MultiWriter(&combinedBuf, &stderrBuf)
+	stdoutWriter := io.Writer(&stdoutBuf)
```

Comment:
This is unnecessary, right?  Buffer is already a Writer.

Reply:
True. Will take that out and write directly to the buffer.